<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieRenderer</class>
 <widget class="QMainWindow" name="MovieRenderer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1572</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VTK Movie Renderer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc/resources.qrc">
    <normaloff>:/MovieRenderer/application.svg</normaloff>:/MovieRenderer/application.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
    <item>
     <widget class="QVTKWidget" name="m_view" native="true">
      <property name="minimumSize">
       <size>
        <width>1280</width>
        <height>720</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1280</width>
        <height>720</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="m_videoGroup">
         <property name="title">
          <string>Video Formats</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="m_renderFull">
            <property name="text">
             <string>Render HD Video</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_renderHalf">
            <property name="text">
             <string>Render 1/2 HD Video</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_rendererGroup">
         <property name="title">
          <string>VTK Renderer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="m_pointSmoothing">
            <property name="text">
             <string>Point Smoothing</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_lineSmoothing">
            <property name="text">
             <string>Line Smoothing</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_polygonSmoothing">
            <property name="text">
             <string>Polygon Smoothing</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_shadows">
            <property name="text">
             <string>Shadows</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
            <item>
             <widget class="QCheckBox" name="m_motionBlur">
              <property name="text">
               <string>Motion Blur</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_motionBlurFrames">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> frames</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
            <item>
             <widget class="QCheckBox" name="m_antiAlias">
              <property name="text">
               <string>AntiAlias</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_aliasFrames">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="suffix">
               <string> frames</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="m_axes">
            <property name="text">
             <string>Show Orientation Marker</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_outputGroup">
         <property name="title">
          <string>Output Directory</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="m_directory">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_dirButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_ffmpegGroup">
         <property name="title">
          <string>FFMPEG </string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="m_ffmpegExe">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_ffmpegDir">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="m_reloadResources">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Reload Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_resetCamera">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Reset Camera</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="m_render">
           <property name="text">
            <string>Render</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_quit">
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>qvtkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
